src/QMCWaveFunctions/SPOSet.cpp

```diff
@@ -109,11 +109,9 @@ void SPOSet::mw_evaluateVGLWithSpin(const RefVectorWithLeader<SPOSet>& spo_list,
                                     const RefVector<ValueVector>& psi_v_list,
                                     const RefVector<GradVector>& dpsi_v_list,
                                     const RefVector<ValueVector>& d2psi_v_list,
-                                    const RefVector<ValueVector>& dspin_v_list) const
+                                    DualMatrix<ComplexType>& mw_dspin) const
 {
-  assert(this == &spo_list.getLeader());
-  for (int iw = 0; iw < spo_list.size(); iw++)
-    spo_list[iw].evaluateVGL_spin(P_list[iw], iat, psi_v_list[iw], dpsi_v_list[iw], d2psi_v_list[iw], dspin_v_list[iw]);
+  throw std::runtime_error(getClassName() + "::mw_evaluateVGLandDetRatioGradsWithSpin() is not supported. \n");
```

Comment:
Need to improve the safe-guard in a later PR.

Reply:
oops, added the wrong name in the error message. Will fix